src/main/java/org/example/HomePage.java

```diff
@@ -0,0 +1,57 @@
+package org.example;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class HomePage extends BasePage {
+    By avatar = By.cssSelector("#userBadge img");
+    By addPlayListButton = By.cssSelector("#playlists i[role='button']");
+    By createPlayListButton = By.cssSelector("[data-testid='playlist-context-menu-create-simple']");
+    By playListNameInput = By.cssSelector("[name='create-simple-playlist-form']>input");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```